about.html

```diff
@@ -0,0 +1,123 @@
+      </div>
+      <p class="invite-msg info-text-about">Please contact us per Email for any further questions about CC Global Summit 2015!</p>
+      <a class="email-about" href="#">ccsummit2015@cckorea.org</a>
+    </section>
+    <section class="logo">
+      <h2 class="secondary-tittle logo-tittle">Creative Commons Global Summit 2015 Logo</h2>
+      <div class="bar-icon"></div>
+      <p class="invite-msg logo-msg">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Minima explicabo inventore omnis necessitatibus molestias labore rerum beatae consequuntur, et eius vitae recusandae, quis quidem. Minus fugit excepturi aut nemo mollitia.</p>
+      <img class="logo-img" src="images/about_logo.png" alt="logo cc Global summit 2015" width="572" height="206">
+    </section>
+    <section class="logo">
+      <h2 class="secondary-tittle logo-tittle">See the past CC Global summits</h2>
+      <div class="bar-icon"></div>
+      <p class="invite-msg logo-msg">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Minima explicabo inventore omnis.</p>
+      <div class="img-past-wrap">
+        <img class="logo-img-past" src="images/about_2013.png" alt="CC Global summit  2013" width="455" height="252">
+        <img class="logo-img-past" src="images/about_2011.png" alt="CC Global summit  2011" width="455" height="252">
+      </div>
+      <p class="invite-msg logo-msg">Lorem ipsum dolor sit amet consectetur adipisicing elit. Nulla voluptatem aut aperiam fugit, modi, iure earum debitis non libero provident tempore? Aliquid maxime itaque dolor labore hic dicta eius dolores. Lorem ipsum dolor sit amet consectetur adipisicing elit. Repellat quod a labore voluptatum suscipit facilis distinctio praesentium ea tempora.</p>
+    </section>
+    <section class="partners sponsor-about">
+      <h2 class="secondary-tittle partner-tittle">Partner</h2>
+      <div class="bar-icon"></div>
+      <div class="partners-wrap">
+        <ul class="partners-list">
+          <li class="partner">
+            <a href="#"><img src="images/mozilla.png" alt="mozzila logo" width="150" height="50"></a>
+          </li>
+          <li class="partner">
+            <a href="#"><img src="images/googlekorea.png" alt="google logo" width="150" height="50"></a>
+          </li>
+          <li class="partner">
+            <a href="#"><img src="images/naver.png" alt="naver logo" width="130" height="50"></a>
+          </li>
+          <li class="partner">
+            <a href="#"><img src="images/kakao.png" alt="Kakao logo" width="120" height="50"></a>
+          </li>
+          <li class="partner">
+            <a href="#"><img src="images/airbnb.png" alt="Airbnb logo" width="130" height="50"></a>
+          </li>
+        </ul>
+      </div>
+    </section>
+  </main>
+  <footer class="about-footer">
```

Comment:
- Please stick to the template provided for the mobile version of the footer. Sticking to the template is a project requirement.
Template mobile footer
<img width="394" alt="Screen Shot 2022-09-23 at 8 26 45 PM" src="https://user-images.githubusercontent.com/47673311/192069057-8511f2ed-0214-47fb-8965-501efccc0e63.png">

Your project mobile footer
<img width="1004" alt="Screen Shot 2022-09-23 at 8 26 08 PM" src="https://user-images.githubusercontent.com/47673311/192069079-5a6eca67-aad0-417b-a8ac-071f79e6878d.png">